Mathlib/CategoryTheory/Limits/Shapes/RegularMono.lean

```diff
@@ -112,14 +112,15 @@ instance MorphismProperty.regularMono.respectsIso :
     (MorphismProperty.regularMono C).RespectsIso :=
   RespectsIso.of_respects_arrow_iso _ (fun _ _ e h ↦ ⟨.ofArrowIso e (h := h.some)⟩)
 
-instance isRegularMono_of_regularMono (f : X ⟶ Y) [h : RegularMono f] : IsRegularMono f := ⟨h⟩
+lemma isRegularMono_of_regularMono {f : X ⟶ Y} (h : RegularMono f) : IsRegularMono f := ⟨h⟩
 
 /-- Given `IsRegularMono f`, a choice of data for `RegularMono f`. -/
 def regularMonoOfIsRegularMono (f : X ⟶ Y) [h : IsRegularMono f] :
     RegularMono f :=
   h.some
 
-instance equalizerRegular (g h : X ⟶ Y) [HasLimit (parallelPair g h)] :
+/-- The chosen equalizer of a parallel pair is a regular monomorphism. -/
+def equalizerRegular (g h : X ⟶ Y) [HasLimit (parallelPair g h)] :
```

Comment:
As this was an instance, there is no real issue changing the name, maybe `RegularMono.equalizer`?